gc/base/NUMAManager.hpp

```diff
@@ -146,6 +152,7 @@ class MM_NUMAManager
 	 */
 	MM_NUMAManager()
 		: _physicalNumaEnabled(false)
+		, _shouldSetCPUAffinity(true)
```

Comment:
This changes the behaviour from default setAffinity, to not setting affinity.  Intentional?

Reply:
The NUMAManager is only used for balanced where I think the GC wants to be setting affinity by default.  I think this change is good but I need to review the OpenJ9 change that is associated with this.

Reply:
@DanHeidinga  `_shouldSetCPUAffinity` is a new field which has been added. We default the value to true because old behavior was to always allow setting the cpu affinity (if numa is available and enabled)